app/code/Magento/AsynchronousOperations/Model/MassSchedule.php

```diff
@@ -139,6 +149,7 @@ public function publishMass($topicName, array $entitiesArray, $groupId = null, $
             try {
                 $operation = $this->operationRepository->createByTopic($topicName, $entityParams, $groupId);
                 $operations[] = $operation;
+                $singleOperationsData[] = $operation->getData();
```

Comment:
To my mind, better is to use DTO from point of view of Service Contracts, than arrays with raw data.